frontend/src/templates/Field/Signature/SignatureField.tsx

```diff
@@ -55,8 +55,13 @@ export const SignatureField = ({
   )
 
   useEffect(() => {
-    formContext.register(schema._id, signatureValidationRules)
-  }, [formContext, schema._id, signatureValidationRules])
+    register(schema._id, signatureValidationRules)
+
+    // when unmounting, unregister the field so that submission will not be blocked by signature validation errors
+    return () => {
+      unregister(schema._id)
```

Comment:
question: have not looked into detail but wondering how the other fields marked as required handle this issue - do they also perform the unregistration? 

asking this as i'm a little cautious as to whether there is an alternative way

Reply:
also wondering - why do we make signature field an uncontrolled element? my understanding is that most fields are controlled and its easier to manage and sync state when it is controlled - hence the recommendation by react docs (source: https://legacy.reactjs.org/docs/uncontrolled-components.html). 

my understanding of React hook form is that in most cases, we want React hook form to control the components (and `Controller` can be used for this purpose https://react-hook-form.com/docs/usecontroller/controller) which would avoid this issue